From 3783ca4476fec8186c867e4e57084e649c318c6b Mon Sep 17 00:00:00 2001
From: Nandor Kollar <nkollar@cloudera.com>
Date: Wed, 10 Jan 2018 14:54:01 +0100
Subject: [PATCH] PARQUET-1185: TestBinary#testBinary unit test fails after
 PARQUET-1141

Author: Nandor Kollar <nkollar@cloudera.com>

Closes #444 from nandorKollar/PARQUET-1185 and squashes the following commits:

533aeb4 [Nandor Kollar] PARQUET-1185: TestBinary#testBinary unit test fails after PARQUET-1141
e75adef [Nandor Kollar] PARQUET-1185: TestBinary#testBinary unit test fails after PARQUET-1141
5e919cb [Nandor Kollar] PARQUET-1185: TestBinary#testBinary unit test fails after PARQUET-1141
---
 .../java/org/apache/parquet/hadoop/thrift/TestBinary.java     | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/parquet-thrift/src/test/java/org/apache/parquet/hadoop/thrift/TestBinary.java b/parquet-thrift/src/test/java/org/apache/parquet/hadoop/thrift/TestBinary.java
index b0d4feb0f2..a152bba620 100644
--- a/parquet-thrift/src/test/java/org/apache/parquet/hadoop/thrift/TestBinary.java
+++ b/parquet-thrift/src/test/java/org/apache/parquet/hadoop/thrift/TestBinary.java
@@ -81,9 +81,9 @@ public void testBinary() throws IOException {
   private void assertSchema(ParquetMetadata parquetMetadata) {
     List<Type> fields = parquetMetadata.getFileMetaData().getSchema().getFields();
     assertEquals(2, fields.size());
-    assertEquals(Types.required(PrimitiveType.PrimitiveTypeName.BINARY).named("s"), fields.get(0));
+    assertEquals(Types.required(PrimitiveType.PrimitiveTypeName.BINARY).id(1).named("s"), fields.get(0));
     assertEquals(OriginalType.UTF8, fields.get(0).getOriginalType());
-    assertEquals(Types.required(PrimitiveType.PrimitiveTypeName.BINARY).named("b"), fields.get(1));
+    assertEquals(Types.required(PrimitiveType.PrimitiveTypeName.BINARY).id(2).named("b"), fields.get(1));
     assertNull(fields.get(1).getOriginalType());
   }
 }
